docs/DeveloperGuide.md

```diff
@@ -164,6 +182,11 @@ method call to return `commandHistory`'s 2nd last command instead of the last co
 With `addToHistory(String command)`, `previousCommand()`, `nextCommand()` and `currentCommand()` implemented, a simple `setOnKeyPressed` under `CommandBox` class which checks
 for user's input of arrow up (which calls previousCommand()) and arrow down (which calls nextCommand()) would suffice for GUI implementation.
 
+Below is the sequence diagram when user pressing the arrow up button with `CommandBox` selected on GUI.
+
+![CommandHistoryTraversalSequenceDiagram](images/CommandHistoryTraversalSequenceDiagram.png)
+
+The previous command by the user will be reflected in the GUI.
```

Comment:
Would you want to explain or provide a brief explanation of the diagram?

Reply:
Added!